orttraining/orttraining/python/training/_ortmodule_training_manager.py

```diff
@@ -20,6 +22,15 @@ class TrainingManager(GraphExecutionManager):
     def __init__(self, model):
         super().__init__(model)
         self._export_mode = torch.onnx.TrainingMode.TRAINING
+    
+    def _get_ort_device_type(self, device):
```

Comment:
Should this be moved to `_utils.py`? There is already a collection of device check/handling helpers there

Reply:
nevermind, removed it.